cartridges/int_twilio/cartridge/scripts/helpers/addToCOHelper.js

```diff
@@ -0,0 +1,42 @@
+
+var Transaction = require('dw/system/Transaction');
+var CustomObjectMgr = require('dw/object/CustomObjectMgr');
+var utils = require('../utils/utils');
+
+
+function addToCOHelper(productId, phoneNumber, productName) {
```

Comment:
As the module is with the same name. You can name it differently, like submit or process.